### DISCUSSION TITLE
How to change language

### DISCUSSION THREAD

**Discussion by blacknet76** [supermerill/SuperSlicer#3199] [Q&A]
Hi at all,

Sorry for this stupid question, but I don't have finde a solution. I want to change the language to German but I can't figure out. 
I cam to this Issue https://github.com/supermerill/SuperSlicer/issues/2227 but I don't find the Slic3r.pot on my ARM Mac.
So I have seen there are some .po files in https://github.com/supermerill/SuperSlicer/tree/master/resources/localization/de but how can I use it?
Can someone please help me?

**Comment by supermerill** [comment]
You change the slicer language by going to Configuration->Language.
To update your language file, you can follow the [doc](https://github.com/supermerill/SuperSlicer/blob/master/resources/localization/README.md)
(compile the po into a mo via gettext, copy & rename it at the right location)

**Comment by blacknet76** [comment]
Thanks for your answer, when I go to Configuration -> Langauge I have only English as selection, there are no other Language to select. 

OK I think there is a problem with the ARM version, I have downloaded the Intel version and there are all Languages available and the SuperSlicer is on startup in german.

**Reply by supermerill** [reply]
oh, I'll check the arm version next time i'll get a mac arm.

**Comment by safademirhan9** [comment]
Is this fixed? I built from source code for Linux and when I run SS, there is only one language option, English. 
I tried following the doc stated above but nothing changed.
